src/drt/src/db/obj/frBTerm.h

```diff
@@ -138,10 +138,7 @@ void frBTerm::serialize(Archive& ar, const unsigned int version)
   (ar) & boost::serialization::base_object<frTerm>(*this);
   (ar) & block_;
   (ar) & pins_;
-  (ar) & net_;
-  if(fr::is_loading(ar) && net_) {
-    net_->addBTerm(this);
-  }
+  // (ar) & net_;
```

Comment:
In bp_fe_top testcase, boost serialization got seg fault because of this. It seems that there is some sort of unresolved deadlock that happens when serializing nets in bterms. So I serialize all the iterms and bterms first, then the nets and connect the terms.

Reply:
A bit worrisome but not gating.